v20.1/architecture/life-of-a-distributed-transaction.md

```diff
@@ -131,7 +131,7 @@ If an operation encounters a write intent for a key, it attempts to "resolve" th
 - `ABORTED`, this operation discards the write intent and reads the next-most-recent value from RocksDB.
 - `PENDING`, the new transaction attempts to "push" the write intent's transaction by moving that transaction's timestamp forward (i.e.,  ahead of this transaction's timestamp); however, this only succeeds if the write intent's transaction has become inactive.
   	- If the push succeeds, the operation continues.
-  	- If this push fails (which is the majority of the time), this transaction goes into the [`TxnWaitQueue`](https:www.cockroachlabs.com/docs/stable/architecture/transaction-layer.html#txnwaitqueue) on this node. The incoming transaction can only continue once the blocking transaction completes (i.e., commits or aborts).
+  	- If this push fails (which is the majority of the time), this transaction goes into the [`TxnWaitQueue`](/docs/stable/architecture/transaction-layer.html#txnwaitqueue) on this node. The incoming transaction can only continue once the blocking transaction completes (i.e., commits or aborts).
```

Comment:
This works, but I think we should just use `transaction-layer.html#txnwaitqueue`, as the target file is in the same directory. 

@anagio, we need to make this same change to other versions of this page.